conda/recipe/meta.yaml

```diff
@@ -66,40 +66,36 @@ requirements:
     - python-clang
   host:
     - python
-    - future >=0.17.1
-    # for python 3.8 the latest protobuf conflicts with other dependencies
-    # leave it as >= 3.18 and let conda just pick the latest that could be resolved
-    # in the dependency tree
-    - protobuf >=3.18.0
-    - libprotobuf-static >=3.18.0
-    - libjpeg-turbo >=2.1.1
+    - future
+    - protobuf
```

Comment:
Hmmm... actually, while touching this one - shouldn't we require <4 ? There were some serious issues with protobuf 4.

Reply:
We link it statically and we are fine with any version. The problem is with TF compiled with one version and used with other.